packages/adapter-utils/src/utils.ts

```
@@ -1152,6 +1152,14 @@ export const getParent = (instance: Element): InstanceElement => {
   return parents[0].value
 }
 
+export const isThereValidParent = (element: Element): boolean => {
+  try {
```

using exceptions for flow control is usually not good practice
I would question the original interface. do we really want something that throws exceptions there rather than returning `undefined`?
it feels like the case being solved in this PR won't be the only place that assumed `getParent` won't throw an exception, and for the code that does except the exception is probably less readable than the usual "or undefined" flow...

---

@ori-moisis Thanks! I'd like this to be included in tomorrow's deployment.
I'll update the interface to return undefined instead of throwing an error in a future PR.